docs/rundown.rst

```diff
@@ -64,6 +102,46 @@ and two small playbooks under a ``playbooks`` directory.
    root of your project. ``ansible-sign project`` commands, as a rule, always
    take the project root directory as their last argument, thus we will simply
    use ``.`` to indicate the current Working Directory.
+Generating a GPG Key
+===================
+
+Full documentation:
+https://docs.github.com/en/authentication/managing-commit-signature-verification/generating-a-new-gpg-key
+
+Follow the link above to download the GPG Command Line Tools.
+
+Once you have the GPG Command line tools, follow these steps to generate your key.
+
+``$ gpg --full-generate-key`` is the command to begin the key generation process.
+
+For most cases, just press enter to accept the default settings. 
+For the expiration date, no expiration date is the default. 
+Once all of that is verified, enter your user ID information. This should be your GitHub email for the email address.
+
+``$ gpg --list-secret-keys --keyid-format=long`` will list the key that has just been created in long form. 
+
+Congratulations! You’re ready to add your key to the Ansible UI.
+
+How to add GPG key to Ansible UI:
+================================
+
+In the command line, run the following commands:
+``$ gpg --list-keys``
+``$ gpg --export --armour <key fingerprint>``
+
+Copy  everything from:
+``-----BEGIN PGP PUBLIC KEY BLOCK-----``
+to 
+``-----END PGP PUBLIC KEY BLOCK-----``
+
+In the Ansible UI, click “Credentials."
+From there, click “Add”.
+For “Name” name your credential something you will recognize.
+For “Credential Type” select  “GitHub Personal Access Token."
+
+Paste what you copied from your command line earlier into the “Type Details” box. Great work! You're ready to start signing content. 
```

Comment:
```suggestion
Paste what you copied from your command line earlier into the “Type Details” box. Great work! You're ready to start verifying signed project content. 
```

but see note above, it's probably better to have them upload the file :slightly_smiling_face: